microsoft-azure/latest/config/index.md

```diff
@@ -200,23 +200,24 @@ The Azure Connector provides a number of properties on installation and for cust
+| connector.az.maxErrorRetries | The maximum number of attempts to retry reads or writes to the Blob container in case of failed transfers. The default is `3`. This configuration uses exponential retries. See [Exponential back-off retry policy](https://docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific#azure-storage){:target="_blank"} for more details. |
 | connector.az.tryTimeout | Indicates the maximum time (in seconds) allowed for any single try of an HTTP request. When transferring large amounts of data, the default tryTimeout will probably not be sufficient. To overcome this, we compute the upload try timeout using: {::nomarkdown}<pre>uploadTryTimeout = tryTimeout * anticipatedPayloadSizeInMb</pre>{:/} |
-| connector.az.objectNamePrefix | You can attach a prefix or suffix when you build the Blob ObjectName from the GUID. This prefix or suffix is not stored in the database, which means correct configuration is needed when using this property. **Note:** The use of this property is strongly discouraged. If you need to separate Blobs from different connectors or systems, then you should use different containers. |
-| connector.az.objectNameSuffix | You can attach a prefix or suffix when you build the Blob ObjectName from the GUID. This prefix or suffix is not stored in the database, which means correct configuration is needed when using this property. **Note:** The use of this property is strongly discouraged. If you need to separate Blobs from different connectors or systems, then you should use different containers. |
-| connector.az.deleted.objectNamePrefix | You can attach a prefix or suffix when you build the Blob ObjectName from the GUID. This prefix or suffix is not stored in the database, which means correct configuration is needed when using this property. **Note:** The use of this property is strongly discouraged. If you need to separate Blobs from different connectors or systems, then you should use different containers. |
-| connector.az.deleted.objectNameSuffix | You can attach a prefix or suffix when you build the Blob ObjectName from the GUID. This prefix or suffix is not stored in the database, which means correct configuration is needed when using this property. **Note:** The use of this property is strongly discouraged. If you need to separate Blobs from different connectors or systems, then you should use different containers. |
+| connector.az.apiEndpoint | Allows configuring a different Azure endpoint. The default value is `blob.core.windows.net`. The value of this property is concatenated with the `connector.az.account.name` property in order to build the final URL, for example, `https://<connector.az.account.name>.<connector.az.apiEndpoint>`. |

```

Comment:
```suggestion
| connector.az.apiEndpoint | Allows configuring a different Azure endpoint. The default value is `blob.core.windows.net`. The value of this property is concatenated with the `connector.az.account.name` property in order to build the final URL, for example, `https://<connector.az.account.name>.<connector.az.apiEndpoint>`. <br><br>Added in Azure Connector 5.0.|
```

Reply:
Added in 4ab82875bb52e93a5d35ad03e07084681ea2bde7 commit.